index.html

```diff
@@ -2241,6 +2246,27 @@ <h1>Core Representations</h1>
 through the content of the <a>DID document</a> alone.
     </p>
 
+    <p>
+Consumers MUST add all properties that do not have explicit processing rules
```

Comment:
```suggestion
Unrecognized properties MUST be preserved. An unrecognized property is
any property that does not have explicit processing rules known to the consumer or producer.
Consumers MUST add all properties that do not have explicit processing rules
```

Reply:
@selfissued this suggestion is an attempt to address your concerns, can you provide feedback so we can move this forward?